css-text/shaping/reference/exp-sub-grapheme-highlight-000-ref.html

```diff
@@ -0,0 +1,111 @@
+<!DOCTYPE html>
```

Comment:
exp-sub-grapheme-highlight-000.html is an exploratory test, so not appropriate for WPT. We also don't need a ref file for it.  So we should remove this file from the set.